src/Core/Managed/Shared/Extensibility/Implementation/OperationTelemetry.cs

```diff
@@ -10,6 +10,8 @@
     /// </summary>
     public abstract class OperationTelemetry : ITelemetry, ISupportMetrics, ISupportProperties
     {
+        internal const string TelemetryName = "Operation";
```

Comment:
I just pushed a change where we _do_ distinguish between ```RequestTelemetry``` and any other ```OperationTelemetry```.  Note that it's possible for a user to create their own operation type derived from ```OperationTelemetry``` (it's public and abstract) and pass that to ```StartOperation<T>```. I don't know how common that is.

Background: Since these operations may be nested, we need to pass the ```EventActivityOptions.Recursive``` flag to ```EventSource.Write```. We could have added that flag to _all_ operations and called it good, but there's a concern that someone will call "StartOperation" without a "StopOperation" leaving a dangling 'start'. It's a catch 22: Either we use "Recursive" everywhere and risk dangling start operations (which can basically ruin analysis of long traces with lots of overlapping operations) or _not_ use recursive, which breaks correlation of Start/Stop events when we have nesting. We compromised by saying ```RequestTelemetry``` should be used at the 'top level' (for incoming requests) and it should not be nested inside any other operation. Other operations _can_ be nested, but users must always be careful to use the ```using``` pattern to guarantee we get a ```StopOperation```.